### PR TITLE
Resize using multiprocessing

### PR DESCRIPTION
**Pull request by bartoszrud**

### CODE REVIEW

File: src/videopython/base/transforms.py [hunk 1]
```diff
@@ -78,15 +79,18 @@ def __init__(self, new_width: int, new_height: int):
         self.new_width = new_width
         self.new_height = new_height
 
+    def _resize_frame(self, frame, new_width, new_height):
```
Comment (BartWojtowicz) [inline src/videopython/base/transforms.py:82]: can we add types? ❤️

File: src/videopython/base/transforms.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 import cv2
 import numpy as np
+from multiprocessing import Pool
```
Comment (BartWojtowicz) [inline src/videopython/base/transforms.py:5]: isn't multiprocessing built-in? 🤔

Comment (bartoszrud) [reply src/videopython/base/transforms.py:5]: It is but you still need to import it

File: src/videopython/base/transforms.py (hunk 1 above)
Comment (bartoszrud) [reply src/videopython/base/transforms.py]: Done!

**Review by BartWojtowicz — APPROVED** [review]
Awesome, LGTM!

**Pull request merged**